adbfs.cpp

```diff
@@ -293,6 +309,30 @@ queue<string> adb_push(const string& local_source,
     return res;
 }
 
+/**
+   Tells Android to rescan the remote file for media changes.
+ */
+queue<string> adb_rescan_file(const string& remote_path)
+{
+    string cmd;
+    cmd.assign("am broadcast -a android.intent.action.MEDIA_SCANNER_SCAN_FILE -d 'file://");
```

Comment:
Apparently, [`ACTION_MEDIA_SCANNER_SCAN_FILE`](https://developer.android.com/reference/android/content/Intent#ACTION_MEDIA_SCANNER_SCAN_FILE) is now deprecated. I've been using [`ACTION_MEDIA_MOUNTED`](https://developer.android.com/reference/android/content/Intent#ACTION_MEDIA_MOUNTED) instead... do you think it's a reasonable alternative?

Reply:
It's not semantically correct, and in principle it could confuse apps which are watching for real mountpoint changes. In practice, it does seem to be abused quite frequently. My personal opinion is that SCAN_FILE is the correct intent and that the deprecation should be viewed as applying only to people writing apps, since the documented replacement is only available to them.

Reply:
Fair enough.